plugins/trace_api_plugin/trace_api_plugin.cpp

```diff
@@ -360,7 +360,7 @@ struct trace_api_plugin_impl {
    :common(common) {}
 
    static void set_program_options(appbase::options_description& cli, appbase::options_description& cfg) {
-      auto cfg_options = cfg.add_options();
+      cfg.add_options();
```

Comment:
Can this line just be entirely removed?

Reply:
Thanks! The whole function `trace_api_plugin_impl::trace_api_plugin_impl` can be removed.